server/src/cli.rs

```diff
@@ -26,19 +25,18 @@ pub fn parse() -> Arguments {
 #[clap( author="The Ankaios team",
         version=env!("CARGO_PKG_VERSION"),
         about="Ankaios - your friendly automotive workload orchestrator.\nWhat can the server do for you?")]
+// default values for the server are set in server-config.rs
 pub struct Arguments {
     #[clap(short = 'c', long = "startup-config")]
     /// The path to the startup config yaml.
     pub path: Option<String>,
-    #[clap(short = 'a', long = "address", default_value_t = DEFAULT_SOCKET_ADDRESS.parse().unwrap())]
-    /// The address, including the port, the server shall listen at.
-    pub addr: SocketAddr,
-    #[clap(
-        short = 'k',
-        long = "insecure",
-        env = "ANKSERVER_INSECURE",
-        default_value_t = false
-    )]
+    #[clap(required = false, short = 'f', long = "server-config")]
```

Comment:
We will use the f for the follow in `ank` and for some tools f could mean force. How about:
```suggestion
    #[clap(required = false, short = 'x', long = "server-config")]
```